torch/_dynamo/variables/torch_function.py

```diff
@@ -639,8 +643,22 @@ def var_getattr(self, tx: "InstructionTranslator", name):
                 [self],
                 {},
             )
-        else:
-            return super().var_getattr(tx, name)
+
+        # Case 3: Check for wrapped torch_function attributes
+        if hasattr(self.source, name):
```

Comment:
I don't understand what this is doing, source.name is a function for most sources, is this code path getting hit in your tests?

Reply:
@mlazos

I see the confusion now. Case 3 might be redundant since
Case 2 already handles torch_function enabled attributes from torch.Tensor. 

My understanding was to handle both:
1. Regular tensor attributes (Case 2)
2. Custom attributes wrapped with @torch.overrides.wrap_torch_function (Case 3)

But perhaps we could simplify this by just checking if an attribute has __torch_function_wrapped__ 
first, and then fall back to the normal torch.Tensor attribute handling?

What do you think would be the right approach to handle custom attributes with torch_function? I saw your previous conversation in the issue discussion but I didn't quite get it